### PR TITLE
Added optech taproot/schnorr library to PoC projects.

### PR DESCRIPTION
**Pull request by jachiang**
Thanks for setting this repository up. I built a Schnorr/MuSig/Taproot library for Optech this summer at the Chaincode residency. The workshop repository which is linked contains a pretty complete Taproot library, with which most of the currently listed PoC projects could be built rather easily. For the purpose of evaluating the Schnorr/Taproot proposals, I suggest participants consider this PoC toolkit first, before investing time to reinvent the wheel?

### CODE REVIEW

File: PoC-Projects.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ These are some ideas for proof of concept projects to explore schnorr and/or tap
  * Taproot/Schnorr support for python-bitcoinlib (etc?)
  * ...
 
+Note: Optech has developed a [Schnorr & Taproot library](https://github.com/bitcoinops/taproot-workshop), which can be used to prototype many of the proof-of-concept projects suggested above. The library features a Schnorr and MuSig implementation as well as methods to construct Taproot outputs from various types of Tapscripts. The documentation is written in the form of interactive Jupyter notebooks, which provide thorough examples on how to use the library.
```
Comment (jnewbery) [inline PoC-Projects.md:12]: @jachiang - optech style guide is to lowercase schnorr and taproot (see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#common-nouns)

**Comment by moneyball** [comment]
fixed this https://github.com/ajtowns/taproot-review/pull/8

File: PoC-Projects.md (hunk 1 above)
Comment (jachiang) [reply PoC-Projects.md:12]: @jnewbery - Just for my understanding, this extends to all following expressions: musig, taproot, taptree, tapscript as well?

**Comment by moneyball** [comment]
the style guidelines say MuSig not musig

**Pull request merged**